website/content/docs/internals/plugins/csi.mdx

```diff
@@ -55,17 +55,25 @@ that perform both the controller and node roles in the same
 instance. Not every plugin provider has or needs a controller; that's
 specific to the provider implementation.
 
-You should always run node plugins as Nomad `system` jobs and use the
-`-ignore-system` flag on the `nomad node drain` command to ensure that the
-node plugins are still running while the node is being drained. Use
-constraints for the node plugin jobs based on the availability of volumes. For
-example, AWS EBS volumes are specific to particular availability zones with a
-region. Controller plugins can be run as `service` jobs.
+Plugins mount and unmount volumes but are not in the data path once
+the volume is mounted for a task. Plugin tasks are needed when tasks
+using their volumes stop, so plugins should be left running on a Nomad
+client until all tasks using their volumes are stopped. The `nomad
+node drain` command handles this automatically by stopping plugin
+tasks last.
+
+Typically, you should run node plugins as Nomad `system` jobs so they
+can mount volumes on any client in a datacenter. Controller plugins
```

Comment:
```suggestion
can mount volumes on any client in a cluster. Controller plugins
```

Unless `datacenter` is intentional (?)

Reply:
It was intentional, because a system job runs across the specific datacenter they're deployed in and not the whole cluster. But on second thought there can be more than 1 DC for a given system job and that makes it read awkwardly. Maybe "can mount volumes on any client where they are running"?